docs/designers-developers/developers/tutorials/notices/README.md

```diff
@@ -0,0 +1,77 @@
+
+Producing an equivalent "Post draft updated" notice would require code like this:
+
+```php
+/**
+ * Hook into the 'admin_notices' action to render
+ * a generic HTML notice.
+ */
+add_action( 'admin_notices', function() {
+	$screen = get_current_screen();
+	// Only render this notice in the post editor.
+	if ( ! $screen || 'post' !== $screen->base ) {
+		return;
+	}
+	// Render the notice's HTML.
+	// Each notice should be wrapped in a <div>
+	// with a 'notice' class.
+	echo '<div class="notice notice-success is-dismissible"><p>';
+	echo sprintf( __( 'Post draft updated. <a href="%s" target="_blank">Preview post</a>' ), get_preview_post_link() );
+	echo '</p></div>';
+} );
+```
+
+Importantly, the `admin_notices` hook allows a developer to render whatever HTML they'd like. One advantage is that the developer has a great amount of flexibility. The key disadvantage is that arbitrary HTML makes future iterations on notices more difficult, if not possible, because the iterations need to accommodate for arbitrary HTML. This is why the Block Editor has a formal API.
+
+## Notices in the Block Editor
+
+In the Block Editor, here's an example of the "Post published" notice:
+
+![Post published in the Block Editor](https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/designers-developers/developers/tutorials/notices/block-editor-notice.png)
+
+Producing an equivalent "Post published" notice would require code like this:
+
+```js
+( function( wp ) {
+	wp.data.dispatch('core/notices').createNotice(
+		'success', // Can be one of: success, info, warning, error.
+		'Post published.', // Text string to display.
+		{
+			isDismissible: true, // Whether the user can dismiss the notice.
+			// Any actions the user can perform.
+			actions: [
+				{
+					url: '#',
+					label: 'View post'
+				}
+			]
+		}
+	);
+} )( window.wp );
+```
+
+You'll want to use this _Notices Data API_ when producing a notice from within the JavaScript application lifecycle.
+
+To better understand the specific code example above:
+
+* `wp` is WordPress global window variable.
+* `wp.data` is an object provided by the Block Editor for accessing the Block Editor data store.
+* `wp.data.dispatch('core/notices')` accesses functionality registered to the Block Editor data store by the Notices package.
+* `createNotice()` is a function offered by the Notices package to register a new notice. The Block Editor reads from the notice data store in order to know which notices to display.
+
+## Learn More
+
+The Block Editor offers a complete API for generating notices. The official documentation is a great place to review what's possible.
+
+For a full list of the available actions and selectors, refer to the [Notices Data Handbook](/docs//designers-developers/developers/data/data-core-notices/) page.
```

Comment:
`For a full list of the available actions and selectors, refer to the [Notices Data Handbook](/docs/designers-developers/developers/data/data-core-notices.md) page.`

Reply:
@mkaz try the changes icon!

![screenshot 2019-02-06 at 18 55 49](https://user-images.githubusercontent.com/1231306/52381783-d2895580-2a40-11e9-80ff-b45c1acaf4e4.png)

Reply:
Ah.... there it is... I've been googling for a few minutes and hadn't figured it out ;-)